.github/workflows/tests.yml

```diff
@@ -24,19 +24,74 @@ jobs:
       - name: checkout ibctest
         uses: actions/checkout@v2
 
-      # cleanup docker environment on self-hosted test runner
-      # - name: prepare fresh docker environment
-      #   run: |
-      #     docker stop $(docker ps -a -q) || true
-      #     docker rm -f $(docker ps -a -q) || true
-      #     docker network prune -f || true
+      # cleanup environment on self-hosted test runner
+      - name: clean
+        run: |-
+          rm -rf ~/.ibctest
+
+      # run tests
+      - name: run unit tests
+        run: go test -race -timeout 30m -v -p 2 ./...
+  test-cmd-ibctest:
+    name: test-cmd-ibctest
+    runs-on: [self-hosted, linux]
+    steps:
+      # Install and setup go
+      - name: Set up Go 1.18
+        uses: actions/setup-go@v2
```

Comment:
There's a setup-go@v3 now. We probably want to use the latest while we're modifying these files. https://github.com/actions/setup-go

Reply:
done